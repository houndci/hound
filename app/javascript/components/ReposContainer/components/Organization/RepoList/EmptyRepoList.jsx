import React from 'react';

export default class EmptyRepoList extends React.Component {
  render() {
    return (<ul className="repos"></ul>);
  }
}
