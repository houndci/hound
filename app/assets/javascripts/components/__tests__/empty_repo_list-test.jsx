import EmptyRepoList from '../empty_repo_list.jsx';

it('renders an empty unordered list', () => {
  const wrapper = shallow(
    <EmptyRepoList />
  );
  expect(wrapper).toMatchSnapshot();
});
