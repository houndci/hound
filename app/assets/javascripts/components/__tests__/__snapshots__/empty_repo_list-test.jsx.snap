exports[`test renders an empty unordered list 1`] = `
<ul
  className="repos" />
`;
